WBooks/Screens/LoginView.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  LoginView.swift
+//  WBooks
+//
+//  Created by Nacho 2 on 14/01/2020.
+//  Copyright © 2020 Wolox. All rights reserved.
+//
+
+import UIKit
+import WolmoCore
+
+class LoginView: UIView, NibLoadable {
+
+    @IBOutlet weak var loginButton: UIButton!
+    
+    @IBAction func loginButtonAction(_ sender: UIButton) {
+        print("anda :D")
+    }
+
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        loginButton.layer.borderWidth = 2.0
+        loginButton.layer.cornerRadius = 20
+        loginButton.layer.borderColor = UIColor.white.cgColor
```

Comment:
You can configure this in `loginButton didSet`.
We have a Wolox-iOS style guide [here](https://github.com/Wolox/ios-style-guide)